packages/ocaml-base-compiler/ocaml-base-compiler.5.0.0~alpha0/opam

```diff
@@ -37,11 +37,17 @@ url {
 }
 extra-files: ["ocaml-base-compiler.install" "md5=3e969b841df1f51ca448e6e6295cb451"]
 post-messages: [
-  "A failure in the middle of the build may be caused by build parallelism
-   (enabled by default).
-   Please file a bug report at https://github.com/ocaml/opam-repository/issues"
-  {failure & jobs > 1}
-  "You can try installing again including --jobs=1
-   to force a sequential build instead."
-  {failure & jobs > 1 & opam-version >= "2.0.5"}
+  "Thank you for testing OCaml 5.0!
+
+   During the release cycle, there can be a lag between packages being fixed for
+   compatibility with the latest release of OCaml and an actual package-release
+   being made to opam-repository. You may wish to add opam-alpha-repository to
+   your switch by issuing:
+
+   opam repo add alpha git+https://github.com/ocaml/opam-alpha-repository
```

Comment:
The url is wrong. The alpha repository is at: https://github.com/kit-ty-kate/opam-alpha-repository
Maybe it should be moved under the ocaml umbrella though

Reply:
Same comment for all the other opam files

Reply:
That was the intention - sorry, I should have been explicit about that in the PR's description!

Reply:
Now that the beta is out and the official guidelines ask to use @kit-ty-kate alpha repo, I think it may be worth just creating its doppleganger in the ocaml org and make this PR mergeable. What do you think?